.automation.conf/config.sh

```diff
@@ -0,0 +1,28 @@
+# This file is used to configure kayobe-automation.
+# https://github.com/stackhpc/kayobe-automation/blob/main/README.md
+
+# See: https://github.com/stackhpc/docker-rally/blob/master/bin/rally-verify-wrapper.sh for a full list of tempest parameters that can be overriden.
+# You can override tempest parameters like so:
+export TEMPEST_CONCURRENCY=2
+# Specify single test whilst experimenting
+#export TEMPEST_PATTERN="${TEMPEST_PATTERN:-tempest.api.compute.servers.test_create_server.ServersTestJSON.test_host_name_is_same_as_server_name}"
+
+KAYOBE_AUTOMATION_TEMPEST_CONF_OVERRIDES="${KAYOBE_AUTOMATION_CONFIG_PATH}/tempest/tempest.overrides.conf"
+
+if [ ! -z ${KAYOBE_ENVIRONMENT:+x} ]; then
+  KAYOBE_AUTOMATION_TEMPEST_CONF_OVERRIDES="${KAYOBE_AUTOMATION_CONFIG_PATH}/tempest/tempest-${KAYOBE_ENVIRONMENT}-${KAYOBE_AUTOMATION_TEMPEST_LOADLIST:-}.overrides.conf"
+
+  # Check if loadlist specific overrides exist, if not fallback to environment overrides.
+  if [ ! -e "${KAYOBE_AUTOMATION_TEMPEST_CONF_OVERRIDES}" ]; then
+      KAYOBE_AUTOMATION_TEMPEST_CONF_OVERRIDES="${KAYOBE_AUTOMATION_CONFIG_PATH}/tempest/tempest-${KAYOBE_ENVIRONMENT}.overrides.conf"
```

Comment:
Just realised we should probably check ${KAYOBE_AUTOMATION_CONFIG_PATH}/tempest/tempest-${KAYOBE_ENVIRONMENT}.overrides.conf exists before setting KAYOBE_AUTOMATION_TEMPEST_CONF_OVERRIDES. That way it falls back to `${KAYOBE_AUTOMATION_CONFIG_PATH}/tempest/tempest.overrides.conf`

Reply:
Done